INSTALL.txt

```diff
@@ -138,18 +160,18 @@ together toolchains for that option. The toolchains are available at
 https://bitbucket.org/carlkl/mingw-w64-for-python/downloads. The site.cfg
 should be configured like so:
```

Comment:
Doh, okay :-). I guess I got confused by the part where all the rest of the file is written in ReST... (???)